### PR TITLE
Create database based on diagram

### PR DESCRIPTION
**Pull request by ZahraArshia**
Hello 😊 
in this step, we created the required database based on the provided diagram of a clinic.  

## Project requirements:

- [x] Create a file named `schema_based_on_diagram.sql` where you implement the database from the diagram. ✅ 

- [x] Join tables from many-to-many relationships might not appear in the diagram, but you still need them. ✅

- [x] Remember to add the FK indexes. ✅

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    total_price decimal,
+    invoice_id int references invoices(id),
+    treatment_id int references treatments(id),
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE medical_documents (
+  medical_histories_id INT references medical_histories(id), 
+  treatments_id INT references treatments(id)
+  );
```
Comment (medaminedev66) [inline schema_based_on_diagram.sql]: - [ ] Following the project requirements, all FOREIGN KEYS should have indexes created. Kindly add the FK indexes.

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
...
+
+
+## Acknowledgments
+
+- Microverse 
+- Coding partner
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (medaminedev66) [inline README.md:46]: - [ ] The MIT link is broken. I would recommend fixing it to make README professional.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @ZahraArshia and @vickymarz 😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- GitFlow best practices have been followed ✔️
- You have a descriptive PR ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (victorokoroji) [reply schema_based_on_diagram.sql]: Thank you for the feedback, we will effect the change ASAP!

File: README.md (hunk 2 above)
Comment (victorokoroji) [reply README.md:46]: Nice observation. Thanks

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi team, 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**